src/Controller/SantaController.php

```diff
@@ -128,18 +141,29 @@ public function message(Rudolph $rudolph, Request $request, string $application)
         $message = $session->get('message');
         $notes = $session->get('notes');
 
+        $form = $this->createForm(MessageType::class, null, [
+            'message' => $message,
```

Comment:
IMO this data should not be passed in the `options` argument but in the `data` argument instead (second argument int the `createForm()` call